src/mds/Server.cc

```diff
@@ -6697,6 +6698,9 @@ void Server::_commit_slave_link(MDRequestRef& mdr, int r, CInode *targeti)
 
   ceph_assert(g_conf()->mds_kill_link_at != 7);
 
+  // now the prepare log can be trimed
+  mdcache->finish_uncommitted_slave(mdr->reqid);
```

Comment:
please also handle other slave update cases.

Reply:
I've also added handle logic for slave_rmdir and slave_rename. But I still can't figure out why we need ESlaveUpdate::OP_COMMIT.  Would you make some explanations? Thanks!